api/src/test/java/org/pmiops/workbench/api/ConceptsControllerTest.java

```diff
@@ -148,6 +160,14 @@ public void testSearchNoConcepts() throws Exception {
             new SearchConceptsRequest().query("a")));
   }
 
+  @Test
+  public void testSearchConceptsMultipleWordQuery(){
+    saveConcepts();
+    assertResults(
```

Comment:
I'm confused -- this seems to be the exact same searchConcepts call as below, with the exception of maxResults(1).  Why doesn't this search return any results when the one below does?

(I can see that both tests are passing... I just don't know why...)

Reply:
Checking the tests.